constants/questionnaire.js

```diff
@@ -188,3 +188,26 @@ export const optionsLogistik = [
     value: 'Tidak, saat ini belum dapat dijangkau kurir logistik'
   }
 ]
+
+export const optionsNotification = [
+  {
+    level: 1,
+    message: 'Desa tidak memiliki akses internet, sehingga belum bisa melanjutkan ke tahap berikutnya. Informasi lebih lanjut, desa Anda akan masuk ke dalam daftar pengajuan penuntasan desa blank-spot untuk bisa ditindaklanjuti.',
+    info: 'Akun Desa Digital Anda akan ditinjau kembali oleh administrator untuk verfifikasi lebih lanjut.'
+  },
+  {
+    level: 2,
+    message: 'Desa masih membutuhkan pelatihan literasi digital, sehingga belum bisa melanjutkan ke tahap berikutnya. Informasi lebih lanjut, desa Anda akan masuk ke dalam daftar kami untuk bisa ditindaklanjuti.',
+    info: 'Akun Desa Digital Anda akan ditinjau kembali oleh administrator untuk verfifikasi lebih lanjut.'
+  },
+  {
+    level: 3,
+    message: 'Desa masih membutuhkan pelatihan terkait pelatihan digital marketing, sehingga belum bisa melanjutkan ke tahap berikutnya. Informasi lebih lanjut, desa Anda akan masuk ke dalam daftar kami untuk bisa ditindaklanjuti. sehatan',
```

Comment:
what does `sehatan` mean? is it a typo?

Reply:
thanks @agunghide, yes it is a typo. and i fix in [here](https://github.com/jabardigitalservice/dedi-landing-page/pull/98/commits/a166595be17da12faf10a56cf896b0a0cd205e4f)